### PR TITLE
[IMP] email marketing: updated unsubscription/blacklist doc for 17

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3763346&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @ksc-odoo,

I can take a look at this.

Thanks,
hojo

File: content/applications/marketing/email_marketing/unsubscriptions.rst [hunk 1]
```diff
@@ -1,66 +1,117 @@
...
+chose to blacklist themselves.
 
 .. image:: unsubscriptions/blacklisted-contact-form.png
    :align: center
    :alt: View of a blacklisted contact detail form in Odoo Email Marketing.
 
-In the :guilabel:`Chatter` of this page, there's a time-stamped message, informing the user when
-that recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).
+In the *chatter* of this blacklisted record page, there's a time-stamped message, informing the user
+when that recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).
```
Comment (hojo-odoo) [inline content/applications/marketing/email_marketing/unsubscriptions.rst:100]:
```suggestion
In the *chatter* of the blacklisted record page, there's a time-stamped message, informing the user
when the recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).

```

File: content/applications/marketing/email_marketing/unsubscriptions.rst [hunk 2]
```diff
@@ -1,66 +1,117 @@
...
 Unblacklist contacts
 ====================
 
-To *Unblacklist* contacts, click the :guilabel:`Unblacklist` button in the upper-left corner to
-remove the contact from the blacklist, allowing them to receive mailings once again.
+To *Unblacklist* contacts, click the :guilabel:`Unblacklist` button in the upper-left corner of a
+blacklisted record's page to remove the contact from the blacklist, allowing them to receive
+mailings once again.
+
+When :guilabel:`Unblacklist` is clicked, a :guilabel:`Are you sure you want to unblacklist this
```
Comment (hojo-odoo) [inline content/applications/marketing/email_marketing/unsubscriptions.rst:109]:
```suggestion
When :guilabel:`Unblacklist` is clicked, an :guilabel:`Are you sure you want to unblacklist this

```

**Review by hojo-odoo — APPROVED** [review]
Hey @ksc-odoo,

Nice job on this doc. I just had a few suggestions. Feel free to move it to the next round of review after you take a look.

Thanks,
hojo

File: content/applications/marketing/email_marketing/unsubscriptions.rst [hunk 3]
```diff
@@ -1,66 +1,117 @@
...
+blacklist themselves.
 
 .. image:: unsubscriptions/blacklisted-contact-form.png
    :align: center
    :alt: View of a blacklisted contact detail form in Odoo Email Marketing.
 
-In the :guilabel:`Chatter` of this page, there's a time-stamped message, informing the user when
-that recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).
+In the *chatter* of the blacklisted record page, there's a time-stamped message, informing the user
+when the recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).
```
Comment (samueljlieber) [inline content/applications/marketing/email_marketing/unsubscriptions.rst:101]: Add a note for more context on what blacklisted emails mean for the database as a whole
```suggestion
.. note::
   Blacklisted emails are excluded from all marketing mailings, however, these emails can still
   receive transactional emails, such as order confirmations, shipping notifications, etc.

```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Awesome job on the improvements in this PR, your changes are looking pretty great! I have a few suggestions for greater clarity and to correct a few points. Please see below, tag me for another quick look once these have been addressed. Thank you!

**Comment by ksc-odoo** [comment]
Alrighty, @samueljlieber -- just made all your suggested changes, and it's ready for another look from you whenever you get a chance. Thanks!

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo! Thanks for implementing my suggestions, this doc is looking good to me! There is one small nitpick I have, if you don't mind addressing:

Update the seealso: doc paths to be relative 🙏
```
.. seealso::
   - :doc:`../email_marketing`
   - :doc:`mailing_lists`
```

I am approving now, and you can tag ZST whenever you are ready. Thank you!

**Comment by ksc-odoo** [comment]
Thanks for the always-helpful feedback, @samueljlieber -- I made that last adjustment, and believe this one is now ready for @StraubCreative 👍

**Review by StraubCreative — APPROVED** [review]
Approved, thanks @ksc-odoo 👍 
...
@robodoo r+

**Pull request closed without merging**